pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -10,7 +10,6 @@
 
 from homeassistant.const import Platform
 
-DEVICE_CLASS = object()
 UNDEFINED = object()
```

Comment:
Sorry for the confusion, I was thinking of something like this
```py
class SentinelValues(StrEnum):
    DEVICE_CLASS = "***DeviceClass"
    UNDEFINED = "***UNDEFINED"
```

Reply:
I've moved UNDEFINED to a sentinel value, but using a sentinel value for DEVICE_CLASS still ends up with:
`W7432: Return type should be [<SentinelValues.DEVICE_CLASS: '***DeviceClass'>, 'str', None] in device_class (hass-return-type)`

Is that better?

Reply:
I managed to get rid of the duplicate on device_class, and this removes the need for the DEVICE_CLASS sentinel.